cav_msgs/msg/DriverStatus.msg

```diff
@@ -28,4 +28,5 @@ bool    comms
 bool    controller
 bool    camera
 bool    imu
+bool    lidar_lite_v3hp
```

Comment:
Rather than using the specific hardware name this should be the driver type. trailer_angle_sensor would be better.